config/environments/production.rb

```diff
@@ -17,42 +14,34 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = false
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
-  config.serve_static_assets = true
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger = ActiveSupport::Logger.new(STDOUT)
-    logger.formatter = config.log_formatter
-    config.logger = ActiveSupport::TaggedLogging.new(logger)
-  end
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
-  config.assets.css_compressor = :sass
+  config.assets.js_compressor = :uglifier
+  # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
-  config.assets.quiet = true # Make static asset logging quiet
 
-  # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  #config.action_controller.asset_host = 'http://wemeditate.co'
-  #config.action_controller.asset_host = 'https://storage.googleapis.com/wemeditate.co'
+  # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
 
-  # More secure csrf tokens
-  config.action_controller.per_form_csrf_tokens = true
```

Comment:
It doesn't seem like this should be removed, but on the other hand I don't think we ever have more than one form on a page in the CMS. But actually, in production it's definitely possible.

Reply:
This became a new default in Rails 5.2 so doesn't have to be specified any more.